src/sentry/static/sentry/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -25,6 +25,7 @@ type Props = AsyncComponent['props'] &
     organization: Organization;
     teams: Team[];
     source?: string;
+    initialData?: InviteRow[];
```

Comment:
Lets do `Partial<InviteRow>[]` and then we can have it merge defaults with whatever the user specifies